packages/config/package.json

```diff
@@ -2,10 +2,14 @@
   "name": "@truffle/config",
   "version": "1.2.0",
   "description": "Utility for interacting with truffle-config.js files",
-  "main": "index.js",
-  "typings": "./typings/index.d.ts",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
   "scripts": {
-    "test": "mocha"
+    "build": "tsc",
+    "prepare": "yarn build",
+    "lint": "tslint src/**/*.ts",
```

Comment:
Thanks for adding these but I'm not sure that we want these extra linting libraries. We already have ESLint that we use project-wide and it looks like tslint will be deprecated this year (see [the repo](https://github.com/palantir/tslint). I don't know, thoughts @gnidan?

Reply:
I deleted it. I think it's better to postpone it. In addition, given that there is already eslint. It remains only to wait for the transition tslint to eslint